src/components/dashboard/PendingTxs/PendingTxListItem.tsx

```diff
@@ -53,36 +18,33 @@ type PendingTxType = {
 const PendingTx = ({ transaction, url }: PendingTxType): ReactElement => {
   return (
     <NextLink href={url} passHref>
-      <a>
-        <StyledContainer>
-          <Grid container py={1} px={2} alignItems="center" gap={1}>
-            <Grid item sx={{ minWidth: '36px' }}>
-              <Typography fontSize="lg" component="span">
-                {isMultisigExecutionInfo(transaction.executionInfo) && transaction.executionInfo.nonce}
-              </Typography>
-            </Grid>
-
-            <Grid item flexGrow={1}>
-              <TxType tx={transaction} />
-            </Grid>
-
-            <TxInfo info={transaction.txInfo} />
-
-            <Grid item xs />
-
-            <TxConfirmations>
-              {isMultisigExecutionInfo(transaction.executionInfo) ? (
-                <StyledConfirmationsCount>
-                  {`${transaction.executionInfo.confirmationsSubmitted}/${transaction.executionInfo.confirmationsRequired}`}
-                </StyledConfirmationsCount>
-              ) : (
-                <Spacer />
-              )}
-              <ChevronRight color="border" />
-            </TxConfirmations>
-          </Grid>
-        </StyledContainer>
-      </a>
+      <Box className={classNames(css.gridContainer, css.columnTemplate)}>
```

Comment:
Is there a reason for removing the `<a>` element here? Now the `div` gets the href and the cursor doesn't change when hovering.

Reply:
Oops good catch! It was not intended